git-cliff/src/lib.rs

```diff
@@ -303,6 +279,36 @@ fn process_repository<'a>(
 			releases.push(Release::default());
 		}
 	}
+	if recurse_submodules {
```

Comment:
I guess we are updating the `releases` vector here based on the commits in the submodules.

It would be nice to extract this to a function that takes `&mut releases` maybe.